<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Shunya</class>
 <widget class="QMainWindow" name="Shunya">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1412</width>
    <height>734</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Shunya</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 rgb(114, 4, 85), stop: 1 rgb(3, 6, 55) )</string>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>577</x>
      <y>240</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 40pt &quot;Consolas&quot;;
background-color: none;
color:white;border:none;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="midLineWidth">
     <number>0</number>
    </property>
    <property name="text">
     <string>WELCOME</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button1">
    <property name="geometry">
     <rect>
      <x>585</x>
      <y>350</y>
      <width>181</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;
border: 0.5px solid white;
border-radius:20px;
background-color:rgba(0, 0, 0,170);
font: 75 18pt &quot;Modern&quot;;</string>
    </property>
    <property name="text">
     <string>S T A R T</string>
    </property>
   </widget>
   <widget class="QLabel" name="image_label">
    <property name="geometry">
     <rect>
      <x>-80</x>
      <y>0</y>
      <width>1491</width>
      <height>750</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button2">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;;background-color:none;border:none;
font: 75 14pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>About us</string>
    </property>
   </widget>
   <widget class="QPushButton" name="settings_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Settings&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color:black;
font: italic 10pt &quot;Segoe UI Semilight&quot;;
border: 0.5px solid white;
border-radius:10px;
background-color: white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button3">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>20</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;;background-color:none;border:none;
font: 75 14pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>Website</string>
    </property>
   </widget>
   <widget class="QLabel" name="_label">
    <property name="geometry">
     <rect>
      <x>-60</x>
      <y>-20</y>
      <width>1471</width>
      <height>750</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="button4">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>20</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;;background-color:none;border:none;
font: 75 14pt &quot;Verdana&quot;;</string>
    </property>
    <property name="text">
     <string>User's Guide</string>
    </property>
   </widget>
   <widget class="QPushButton" name="insta_btn">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>630</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Instagram&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: italic 10pt &quot;Segoe UI Semilight&quot;;
border-radius:10px;
 background-color: QLinearGradient( x0: 1, y0: 0, x2: 0, y2: 1, stop: 0 rgb(254,218,117), stop: 1 rgb(214,41,118) )</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="ln_btn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>630</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;LinkedIn&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: white;
font: italic 10pt &quot;Segoe UI Semilight&quot;;
border-radius:10px;
background-color: rgb(0, 119, 181);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>600</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Nirmala UI&quot;;
color:white;background-color:none;border:none;</string>
    </property>
    <property name="text">
     <string>Follow us:</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="dt">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>680</y>
      <width>1271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:transparent;
color:white;
border:none;</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;yudgf&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close_btn">
    <property name="geometry">
     <rect>
      <x>1280</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exit&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;
font: italic 10pt &quot;Segoe UI Semilight&quot;;
border-radius:10px;
background-color: black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>_label</zorder>
   <zorder>image_label</zorder>
   <zorder>label</zorder>
   <zorder>button1</zorder>
   <zorder>button2</zorder>
   <zorder>settings_button</zorder>
   <zorder>button3</zorder>
   <zorder>button4</zorder>
   <zorder>insta_btn</zorder>
   <zorder>ln_btn</zorder>
   <zorder>label_2</zorder>
   <zorder>dt</zorder>
   <zorder>close_btn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
